<launch>
    <!-- <arg name="control_command_delay_" default="0.05"/> -->
    <node pkg="mpc_pkg" type="ibvs_mpc1_controller_node" name="ibvs_mpc1_controller_node" output="screen" >
        <rosparam file="$(find mpc_pkg)/parameters/default.yaml" />
    </node>
</launch>